### ISSUE TITLE
Dismiss NavigationView

### ISSUE THREAD

**Issue by ivedeneev** [rundfunk47/stinsen#39]
Hi! Im presenting coordinator like this
```
@Route(.modal) var card = makeCard
    
func makeCard() -> NavigationViewCoordinator<CardCoordinator> {
    NavigationViewCoordinator(CardCoordinator())
}
```
Card coordinator contains 3 screens in navigation stack.
Then im trying to dismiss it like this in last view in navigation stack:
```
router.dismissCoordinator {
      print("dismiss")
}
```

After that im facing crash in `dismissChild` method of `NavigationCoordinatable`

How can i dismiss it correctly?

**Comment by rundfunk47** [comment]
Hi! It looks like we have a bug in the latest version of Stinsen. Try downgrading for the moment and see if it solves your issue :)

**Comment by rundfunk47** [comment]
Should be fixed in 2.0.5. Please try it and see if it solves your issue.

**Comment by ivedeneev** [comment]
It works now, thanks!
